CMakeLists.txt

```diff
@@ -14,7 +14,11 @@ if(BOOST_SUPERPROJECT_VERSION)
     set(BOOST_JSON_VERSION ${BOOST_SUPERPROJECT_VERSION})
 endif()
 
-project(boost_json VERSION ${BOOST_JSON_VERSION} LANGUAGES CXX)
+if (BOOST_SUPERPROJECT_VERSION)
```

Comment:
Isn't this pointless given https://github.com/CPPAlliance/json/pull/60/files#diff-af3b638bc2a3e6c650974192a53c7291R13-R15

Reply:
Sorry, no idea what this is referring to, lines 13-15 are

```
## Introduction

This library provides containers and algorithms which implement JSON
```

Reply:
Those lines are in the CML, Github doesn't show it by default as it is not part of the changeset AND jumps to the wrong file -.-:

```
if(BOOST_SUPERPROJECT_VERSION)
    set(BOOST_JSON_VERSION ${BOOST_SUPERPROJECT_VERSION})
endif()
```

Reply:
> Isn't this pointless given

🤷‍♂ ask @vinniefalco 

The `BOOST_SUPERPROJECT_VERSION` handling was there before and I did not aim to remove it. I only patched it a bit.

Reply:
Yeah but the original version already took care of BOOST_SUPERPROJECT_VERSION and https://github.com/CPPAlliance/json/pull/60/commits/bbfb09b26c8acb105c02328a9222af2af384d94f added superflous BOOST_SUPERPROJECT_VERSION handling. So I'd just revert that commit. The original is more DRY

Reply:
Oh, good catch!

Not sure what I was thinking about or it was left after some numerous experiments and CMakeLists.txt edits. Done in #62